src/NzbDrone.Core/Languages/Language.cs

```diff
@@ -102,6 +102,7 @@ public override bool Equals(object obj)
         public static Language Malayalam => new Language(29, "Malayalam");
         public static Language Ukrainian => new Language(30, "Ukrainian");
         public static Language Slovak => new Language(31, "Slovak");
+        public static Language Original => new Language(-2, "Original");
```

Comment:
Why `-2` instead of `-1`, what is `-1`?

Reply:
In Radarr `-1` is `Any` which becomes obsolete when we drop it off of the quality profile to be consistent with Sonarr. This was to make CFs between the two consistent, but can use `-1` here if we want.

Reply:
Seems reasonable to keep it the same, I was just wondering why it was `-2`.... `Any` is a separate language though... WTF.